impl/src/main/java/org/glassfish/soteria/identitystores/LdapIdentityStore.java

```diff
@@ -185,10 +185,20 @@ private CredentialValidationResult checkThroughSearch(UsernamePasswordCredential
                         null, // caller unique id
                         groups);
 
-            } catch (IllegalStateException e) {
-                return NOT_VALIDATED_RESULT;
+            } catch (Exception e) {
+                if (e instanceof AuthenticationException) {
+                  LOGGER.log(Level.SEVERE, "Error validating credential " + e);
```

Comment:
Not sure we want to log errors user credentials. If it's a config problem, we'll want to indicate that. Same comment for each of the exceptions we're catching.